test/ftp_object.jl

```diff
@@ -66,6 +66,13 @@ end
     close(ftp)
 end
 
+# @testset "connection with url" begin
+#     host = hostname(server)
+#     ftp = FTP(; url="ftp://$host/", opts...)
+#     @test ftp.ctxt.url == "ftp://$host/"
+#     close(ftp)
+# end
+
```

Comment:
Is the test supposed to be commented out?

Reply:
Oops!